### PR TITLE
update script to update version after merge

### PR DESCRIPTION
**Pull request by kmathew-pdftron**

### CODE REVIEW

**Review by sgong-pdftron — COMMENTED** [review]
code looks fine for me, did you test and see if it works?

**Comment by kmathew-pdftron** [comment]
yes it works

File: .github/workflows/npm_publish.yml [hunk 1]
```diff
@@ -27,7 +27,25 @@ jobs:
...
       - run: npm install json    
-      - run: npm publish --access public
+
+      - name: Publish Package
+        run: npm publish --access public
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_AUTH_TOKEN}}
+
+      - name: add json file changes
+        run: git add  package.json  # <-- Change this to your build path.
```
Comment (sgong-pdftron) [inline .github/workflows/npm_publish.yml:39]: there is strange spacing

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]
I still don't understand how this works, because isn't this going to create 2 separate commit in master? one for the PR, one for the version change? ideally we want 1 commit for both

**Review by sgong-pdftron — APPROVED** [review]

**Pull request merged**